packages/tadviewer/src/components/CellClickData.tsx

```diff
@@ -0,0 +1,8 @@
+export interface CellClickData {
+  readonly value: any;
+  readonly cell: { readonly row: number; readonly col: number };
+  readonly column: {
+    readonly displayName: string;
+    readonly columnType: string;
+  };
```

Comment:
My one (and hopefully final) request here:
Tad works mainly in terms of `columnId` (of type `string`), which is a SQL identifier that can be used in a `select` clause to extract the column.
Could you please include the Tad `columnId` in the `column` object in `CellClickData`?